packages/lib/modules/vebal/lock/steps/useLockStep.tsx

```diff
@@ -102,6 +99,8 @@ export function useLockStep({ lockAmount, lockEndDate, lockActionType }: UseLock
       onActivated: () => setIsStepActivated(true),
       onDeactivated: () => setIsStepActivated(false),
       renderAction: () => <ManagedTransactionButton id={lockActionType.toString()} {...props} />,
+      // only used for integration testing
```

Comment:
This comment can also be on the type definition just in case someone checks it there.